docs/stations/SNZO_IU_Wellington_station.md

```diff
@@ -0,0 +1,62 @@
+## General Information 
+
+**Station Information SNZO** - Location: South Karori, Wellington, New-Zealand
+
+Present in Delta GeoNet  station management as an Auxilliary station in mainland NZ
+
+Foreign Agency ownership: Network Affiliation: IRIS/USGS - GSN (US) 
+DOI: https://doi.org/10.7914/SN/IU
+  
+**Context/history**
+- Seismic station from the Global Seismic Network (GSN) (United States)
+Started in 1975 by the Seismic Research Observatory 
+Used for global seismology
+
+- 24 August 2024: SNZO station and streams are in part set
+for being an auxilliary seismic station for NZ monitoring.
+
+SNZO streams (NZ settings) are used as any continuous real-time station from the NZ National Network 
+
+**Equipement** 
+
+As used by Geonet (2024):
+  - Broadband Borehole Velocimeter
+  - Broadband Borehole Accelerometer
+  
+Other instruments are not described in Delta. (e.g. BB, VBB and barometers) 
+
+## Detailed Information 
+
+### Data
+- **IU SNZO stream settings for IU**
+Meant for global survey - Has low Real-time stream sampling (weak and strong) and barometers
+
+- **IU SNZO stream settings for NZ, GeoNet**
+Derived for local/regional monitoring, has 100 Hz real-time weak and  200 Hz strong motion. 
```

Comment:
What are the channel codes?

Reply:
HH and HN continuous  shall i add this ? @staylorofford

Reply:
If you want, could be handy at some point